R/zchunk_LB141.ag_Fert_IFA_ctry_crop.R

```diff
@@ -40,43 +42,259 @@ module_aglu_LB141.ag_Fert_IFA_ctry_crop_DISABLED <- function(command, ...) {
     IFA2002_Fert_ktN <- get_data(all_data, "aglu/IFA2002_Fert_ktN")
     IFA_Fert_ktN <- get_data(all_data, "aglu/IFA_Fert_ktN")
 
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
+
+    # Perform Calculations
+
+    # Lines 65-69 in original file
+    # Convert IFA fertilizer consumption to long form and convert units to Mt of N consumed.
+    # This is top down fertilizer consumption data, to be reconciled with bottom up consumption
+    # estimate using IFA2002 fertilizer consumption, and LDS, FAO and Monfreda harvested area data.
+    IFA_Fert_ktN %>%
+      filter(Country != "World Total") %>%
+      gather(IFA_commodity, value, -Country) %>%
+      # convert units from kt to Mt of Nitrogen
+      mutate(Fert_MtN = value * CONV_KT_MT) %>%
+      # rename Country
```

Comment:
For what it's worth, I personally find this kind of comment not helpful; it takes up screen real estate, and doesn't tell us anything more than a glance at the code would. You can and should assume basic (programming) literacy on the part of the reader, I think.